tests/utils.ts

```diff
@@ -90,6 +90,25 @@ export async function callContract(
   await sendAndReturnFinalized(signer, tx);
 }
 
+export async function rpcContract(
+  api: ApiPromise,
+  dest: Address,
+  inputData: any,
+  gasLimit: number = 2147483648,
```

Comment:
Could you use `GAS_REQUIRED` here. The naming is not quite correct anymore after some changes in the contracts pallet. You can also rename it to `GAS_LIMIT`, if that feels better.

Reply:
GAS_REQUIRED does not work, I get a message saying it does not fit in a u32. I'll create a GAS_LIMIT.

Reply:
Interesting, I thought it changed to u64 altogether...

Reply:
That would make sense, however current substrate is not accepting it. I've just pulled latest master in substrate and says:
    2020-06-02 17:09:27        RPC-CORE: call(callRequest: ContractCallRequest, at?: BlockHash): ContractExecResult:: -32602: `100000000000` > u32::max_value(), the max block number is u32.
This probably a bug in @polkadot/api-contract.